src/main/java/ru/skypro/courseWork/controller/AdController.java

```diff
@@ -0,0 +1,97 @@
+package ru.skypro.courseWork.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import ru.skypro.courseWork.dto.*;
+
+@CrossOrigin(value = "http://localhost:3000")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/ads")
+@Tag(name = "Объявления")
+public class AdController {
+
+    @GetMapping
+    @Operation(summary = "Получение всех объявлений", responses = {
+            @ApiResponse(responseCode = "200", description = "OK", content = {
+                    @Content(mediaType = "application/json", array = @ArraySchema(schema = @Schema(implementation = AdsDto.class)))})
+    })
+    public ResponseEntity<AdsDto> getAllAds() {
+        return ResponseEntity.ok(new AdsDto());
+    }
+
+    @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    @Operation(summary = "Добавление объявления", responses = {
+            @ApiResponse(responseCode = "201", description = "Created", content = {
+                    @Content(mediaType = "application/json", array = @ArraySchema(schema = @Schema(implementation = AdDto.class)))}),
+            @ApiResponse(responseCode = "401", description = "Unauthorized")
+    })
+    public ResponseEntity<AdDto> createAd(@RequestBody CreateOrUpdateAdDto properties, @RequestParam MultipartFile image) {
```

Comment:
```suggestion
    public ResponseEntity<AdDto> createAd(@RequestParam CreateOrUpdateAdDto properties, @RequestParam MultipartFile image) {
```
вот так должно быть. Или вот так можно тоже
```suggestion
    public ResponseEntity<AdDto> createAd(@RequestPart CreateOrUpdateAdDto properties, @RequestPart MultipartFile image) {
```